bramkouwen/Rock-Paper-Scissors-Game
# Updating game UX/UI

## What does this PR do?
* It Updates game.py to have more of a user interface

## How do I check this PR?
* Pull down the branch 
* See UX/UI changes to game.py

## Screenshots 
None.
